workflow/scripts/gen_new_features.py

```diff
@@ -20,8 +20,10 @@ def add_midpoint(df):
 @click.option("--ref_gene_tss")
 @click.option("--chr_sizes")
 @click.option("--results_dir")
-def main(enhancer_list, abc_predictions, ref_gene_tss, chr_sizes, results_dir):
-    intermediate_dir = create_intermediate_dir(results_dir)
+@click.option("--dataset")
```

Comment:
dataset is only relevant for output directory right? If so, we can just modify the input s.t the dataset is part of `results_dir`. e.g `results_dir = results/{dataset}`